packages/design-system/src/__theme__/default/index.css

```diff
@@ -1,133 +1,116 @@
 @import "colors.css";
 @import "variables.css";
 @import "typography.css";
+@import "semantic.css";
 
 :root {
   /**
     * ===========================================*
-    * ========== Background Colors ==============*
+    * =========== Category Tokens ===============*
     * ===========================================*
   */
 
-   /** Used as active bg when --ads-v2-color-bg-brand is the default bg **/
-  --ads-v2-color-bg-brand-emphasis-plus: var(--ads-v2-color-orange-800);
-  /** Used as hover bg when --ads-v2-color-bg-brand is the default bg **/
-  --ads-v2-color-bg-brand-emphasis: var(--ads-v2-color-orange-700);
-  /** Brand primary color **/
-  --ads-v2-color-bg-brand: var(--ads-v2-color-orange-600);
-
-  /** Used as active bg when --ads-v2-color-bg-brand-secondary is the default bg */
-  --ads-v2-color-bg-brand-secondary-emphasis-plus: var(--ads-v2-color-gray-600);
-  /** Used as hover bg when --ads-v2-color-bg-brand-secondary is the default bg */
-  --ads-v2-color-bg-brand-secondary-emphasis: var(--ads-v2-color-gray-700);
-  /** Brand secondary color */
-  --ads-v2-color-bg-brand-secondary: var(--ads-v2-color-gray-600);
-
-  /** Used for tooltips or any high contrast components */
-  --ads-v2-color-bg-emphasis-max: var(--ads-v2-color-gray-800);
-  /** Used for hover state for components without border */
-  --ads-v2-color-bg-emphasis-plus: var(--ads-v2-color-gray-400);
-  /** Used for components without border */
-  --ads-v2-color-bg-emphasis: var(--ads-v2-color-gray-300);
-  /** Default BG */
-  --ads-v2-color-bg: var(--ads-v2-color-white);
-  /* Used for emphasis default bg */
-  --ads-v2-color-bg-muted: var(--ads-v2-color-gray-200);
-  /** Used for disabled components */
-  --ads-v2-color-bg-subtle: var(--ads-v2-color-gray-100);
-
-  /** Used for error background active */
-  --ads-v2-color-bg-error-emphasis-plus: var(--ads-v2-color-red-300);
-  /** Used for error background hover  */
-  --ads-v2-color-bg-error-emphasis: var(--ads-v2-color-red-200);
-
-  /** Used for error background */
-  --ads-v2-color-bg-error: var(--ads-v2-color-red-100);
-  /** Used for success background */
-  --ads-v2-color-bg-success: var(--ads-v2-color-green-100);
-  /** Used for warning background */
-  --ads-v2-color-bg-warning: var(--ads-v2-color-yellow-100);
-  /** Used for information background */
-  --ads-v2-color-bg-information: var(--ads-v2-color-blue-100);
-
-  /**
-    * ===========================================*
-    * ========== Foreground Colors ==============*
-    * ===========================================*
+  /* Naming convention: 
+    *
+    * (Atom-Category-Component-Variant-Surface-State-PresenationalAttribute)
+    * Atom - Type of token (colors).
+    * Category - Indicates which category the token belongs to (action, response, control etc).
+    * Component - Indicates which component the token belongs to (button, input, select etc).
+    *             this should be avoided as much as possible.
```

Comment:
Instead of `this should be avoided as much as possible.`, write about under what circumstances this can be done, with example (link and avatar)